test/WebSites/OutOfProcessWebSite/web.config

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0"?>
-<configuration>
```

Comment:
Still on the fence about whether web.configs should be removed or not.

Reply:
It's required to avoid conflicts.